src/lib/containers/entity_finder/details/view/DetailsView.tsx

```diff
@@ -68,7 +70,6 @@ export type DetailsViewProps = EntityDetailsListSharedProps & {
 export type DetailsViewRowData = (EntityHeader | ProjectHeader | Hit) & {
   entityType: EntityType
   isSelected: boolean
-  isHidden: boolean
```

Comment:
Hidden items are now excluded from RowData, so this property is no longer needed.